dev/sandbox.clj

```diff
@@ -1,5 +1,7 @@
 (ns sandbox
-  (:require [datahike.api :as d]))
+  (:require [datahike.api :as d]
+            [datahike-leveldb.core]
+            [datahike-postgres.core :as pg]))
```

Comment:
Are both `backends` always needed here if we remove them in the future?

Reply:
sorry, I guess the sandbox was a mistake.

Reply:
Great, thanks for the refactoring.